Resources/input/fsh/ISiKBookOperation.fsh

```diff
@@ -41,7 +41,7 @@ Usage: #example
   * use = #out
   * min = 0
   * max = "1"
-  * documentation	= "Falls die Interaktion erfolgreich ist, ist der Status Code HTTP 201 - Created zurückzugeben. Zudem MUSS Als Return-Parameter entweder eine Appointment-Ressource (entsprechend dem dazugehörigen ISiK-Profil) oder eine Parameters-Ressource mit mindestens einem .parameter-Element Appointment (entsprechend dem ISiK-Profil) und einem .parameter-Element OperationOutcome zurückgegeben werden und die Appointment-Ressource MUSS verarbeitet werden (eine Parameters-Ressource mit einem einzigen .parameter-Element 'Appointment' sollte nicht zurückgegeben werden). Das id-Element der Ressource Appointment MUSS korrekt gefüllt werden. Der Status der Appointment-Ressource MUSS auf 'booked' oder 'pending' geändert werden. Der Server MUSS die verwendeten Slot-Ressourcen als Referenz im Appointment angeben."
+  * documentation	= "Falls die Interaktion erfolgreich ist, ist der Status Code HTTP 201 - Created zurückzugeben. Im Erfolgsfall MUSS als Return-Parameter eine Appointment-Ressource (entsprechend dem dazugehörigen ISiK-Profil) zurückgegeben werden. Das id-Element der Appointment-Ressource MUSS korrekt gefüllt werden. Der Status der Appointment-Ressource MUSS auf 'booked' oder 'pending' geändert wurden sein durch den Server. Der Server MUSS die verwendeten Slot-Ressourcen als Referenz im Appointment angeben. Die Appointment-Ressource kann direkt im HTTP Body zurückgegeben werden, falls es sich um den einzigen Rückgabewert handelt. Alternativ kann die Appointment-Ressource innerhalb einer Parameter-Ressource zurückgegeben werden. Als Teil der Parameter-Ressource können zusätzlich Hinweise zum Termin als OperationOutcome zurückgegeben werden."
```

Comment:
Auch hier eher 'SOLLTE' oder eigentlich sogar 'MUSS' statt 'KANN': "Die Appointment-Ressource kann direkt im HTTP Body zurückgegeben werden, falls es sich um den einzigen Rückgabewert handelt."
Siehe Core-spec hier: "If there is a single output parameter named 'return' with a maximum cardinality of 1, with a type corresponding to a resource, then the response SHALL be the resource that is the return value, with no wrapping Parameters resource." (https://build.fhir.org/operations.html)

Reply:
Der Satz ist neu in R5! Guter Hinweis, dann könnten wir es tatsächlich strenger spezifizieren.

Reply:
Ich wäre auch für das SOLLTE